WithSecure/withsecure-elements/ingest/parser.yml

```diff
@@ -73,6 +80,15 @@ stages:
           withsecure.file.rarity: "{{json_event.message.details.rarity}}"
           withsecure.recommended_action: "{{json_event.message.details.recommandedAction}}"
 
+      - set:
+          user.name: "{{json_event.message.userName}}"
+          user.domain: "{{json_event.message.userName}}"
```

Comment:
My bad. Can you split the content of `userName` with the first part for `user.domain` and the second part in `user.name` when `userName` contains `\` or `//`?

Reply:
Fixed thank you!